samples/customchecks/monitoring3rdparty/Version_3_3/3rdPartySystem/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```

Comment:
but you are not trying to demonstrate web api so familiarity with it is a not an issue. and in this case we are trying to simulate an external web service that they dont own so we should choose the lightest weight approach so as to not confuse the user